src/pageTypes/views/PageTypeList/PageTypeList.tsx

```diff
@@ -118,6 +125,41 @@ export const PageTypeList: React.FC<PageTypeListProps> = ({ params }) => {
 
   const handleSort = createSortHandler(navigate, pageTypeListUrl, params);
 
+  const [
+    pageTypeBulkDelete,
+    pageTypeBulkDeleteOpts
+  ] = usePageTypeBulkDeleteMutation({
+    onCompleted: data => {
+      if (data.pageTypeBulkDelete.errors.length === 0) {
+        notify({
+          status: "success",
+          text: intl.formatMessage(commonMessages.savedChanges)
+        });
+        reset();
+        refetch();
+        navigate(
+          pageTypeListUrl({
+            ...params,
+            action: undefined,
+            ids: undefined
+          })
+        );
+      }
+    }
+  });
+
+  const hanldePageTypeBulkDelete = () =>
+    pageTypeBulkDelete({
+      variables: {
+        ids: params.ids
+      }
+    });
+
+  const selectedPageTypesHasPages = data?.pageTypes.edges.some(
+    pageType =>
+      pageType.node.hasPages && params.ids?.some(id => id === pageType.node.id)
```

Comment:
Is second check necessary?

Reply:
Yes because we want to check only those page types which are selected.